src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -527,6 +542,7 @@ protected function getMediaData($media)
                     $fileName = $fileVersion->getName();
                     $storageOptions = $fileVersion->getStorageOptions();
```

Comment:
the concept was that the storageOptions could also be a serialized class or just a string and only the specific storage know how to handle it. but maybe we should force that the storageOptions are an array which make it easier to use.
what you think @danrot @wachterjohannes